recipes/tar/all/conanfile.py

```diff
@@ -39,10 +43,15 @@ def package_id(self):
     def requirements(self):
         self.requires("bzip2/1.0.8", run=True, headers=False, libs=False)
         self.requires("lzip/1.23", run=True, headers=False, libs=False)
-        self.requires("xz_utils/5.4.4", run=True, headers=False, libs=False)
+        self.requires("xz_utils/5.4.5", run=True, headers=False, libs=False)
         self.requires("zstd/1.5.5", run=True, headers=False, libs=False)
         # self.requires("lzo/2.10", run=True, headers=False, libs=False)
 
+    def build_requirements(self):
+        if Version(self.version) == "1.35":
```

Comment:
Is this not expected to be needed in the future? We might save a compilation if the check in the future includes the new versions :)

Reply:
This is only needed because autoreconf needs to be called after patching a Makefile.am file.
Given the patch comes from the main branch of tar, it's not expected that this will be needed in the future.

Reply:
@RubenRBS, do you need more information ?

Reply:
No, sorry I missed this ping :/